CONTRIBUTING.md

```diff
@@ -36,7 +36,7 @@ This project adheres to the American Express [Code of Conduct](./CODE_OF_CONDUCT
 
 4. You can now run any of these scripts from the root folder.
 
-#### Running tests
+#### Running scripts
```

Comment:
```suggestion
#### Running test scripts
```

Reply:
these are npm scripts, so i thought scripts would be more relevant than test scripts.

Reply:
scripts that define the tests for the package :)